src/crypto/ocb_internal.cc

```diff
@@ -360,21 +360,43 @@
 
 #include <openssl/aes.h>                            /* http://openssl.org/ */
 
+namespace ocb_aes {
+
+typedef AES_KEY KEY;
+
+static void set_encrypt_key(const unsigned char *user_key, int bits, KEY *key) {
+	AES_set_encrypt_key(user_key, bits, key);
+}
+
+static void set_decrypt_key(const unsigned char *user_key, int bits, KEY *key) {
+	AES_set_decrypt_key(user_key, bits, key);
+}
+
+static void encrypt(const unsigned char *in, unsigned char *out, KEY *key) {
+	return AES_encrypt(in, out, key);
+}
+
+static void decrypt(const unsigned char *in, unsigned char *out, KEY *key) {
+	return AES_decrypt(in, out, key);
+}
+
 /* How to ECB encrypt an array of blocks, in place                         */
-static inline void AES_ecb_encrypt_blks(block *blks, unsigned nblks, AES_KEY *key) {
+static inline void ecb_encrypt_blks(block *blks, unsigned nblks, KEY *key) {
 	while (nblks) {
 		--nblks;
-		AES_encrypt((unsigned char *)(blks+nblks), (unsigned char *)(blks+nblks), key);
+		encrypt((unsigned char *)(blks+nblks), (unsigned char *)(blks+nblks), key);
```

Comment:
Do we want to switch to C++-style casts while we're here? I believe we are compiling this file in C++ mode.

Reply:
Done.